util/gdas_init/config

```diff
@@ -52,16 +52,16 @@ EXTRACT_DATA=no
 
```

Comment:
Why was this file changed?

Reply:
Oh, I was probably testing something for another PR and it was inadvertently committed. I just reverted it.